catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java

```diff
@@ -465,96 +508,115 @@ private boolean processBatch(CatalogFacade catalog, ArrayList<Metacard> metacard
     }
 
     private void moveToFailedIngestDirectory(File source) {
-        File destination = new File(failedIngestDirectory.getAbsolutePath() + File.separator + source.getName());
+        File destination = new File(
```

Comment:
Yeah. I hadn't done a lot of cleanup within the methods that were already there. Should have taken the extra time.